macro/run_simScript.py

```diff
@@ -136,13 +136,7 @@
 parser.add_argument("--scName", help="The name of the SC shield in the database", default="sc_v6")
 parser.add_argument("--MesonMother",   dest="MM",  help="Choose DP production meson source", required=False,  default=True)
 parser.add_argument("--debug",  help="1: print weights and field 2: make overlap check", required=False, default=0, type=int, choices=range(0,3))
-parser.add_argument(
-     "--helium",
-     help="Replace vacuum with helium (vessel structure unchanged)",
-     action="store_const",
-     const="helium",
-     default="vacuums"
-)
+parser.add_argument("--vacuums",dest="decayVolMed",help="Replace Decay Volume medium from helium to vacuum (vessel structure changes)",action="store_const",const="vacuums",default="helium")
```

Comment:
It would be good to keep the `--helium` flag around for now as a NOOP, to not break the interface.

Could you please format this line sensibly?